Python/Product/Analysis/Interpreter/Ast/AstPythonFunction.cs

```diff
@@ -54,6 +54,17 @@ LocationInfo loc
             Locations = loc != null ? new[] { loc } : Array.Empty<LocationInfo>();
         }
 
+        internal AstPythonFunction(IPythonFunction original) {
+            DeclaringModule = original.DeclaringModule;
+            DeclaringType = original.DeclaringType;
+            Name = original.Name;
+            _doc = (original as AstPythonFunction)?._doc ?? original.Documentation;
```

Comment:
`AstPythonFunction.Documentation` first checks for `_doc` as well, no need to check for it separately.

Reply:
I meant to copy the `null` from `_doc`, but this doesn't do that either :) So a slightly different change coming